MANIFEST.in

```diff
@@ -1,3 +1,4 @@
 include LICENSE.md
+include uvicorn/default_logging_config.json
```

Comment:
Alternatively, consider adding

```python
include_package_data=True,
```

 to `setup.py` (if not present already), and putting the JSON file under `uvicorn/data/...`?

Then in `MANIFEST.in`:

```
graft uvicorn
```

Reply:
Good point to store the config outside of the source. I've created the `data` directory